airflow-core/src/airflow/ui/public/i18n/locales/zh-CN/hitl.json

```
@@ -23,6 +25,7 @@
     "approvalRequired": "需要批准",
     "choiceReceived": "已选择",
     "choiceRequired": "需要选择",
+    "noResponseReceived": "需要操作的",
```

```suggestion
    "noResponseReceived": "未收到响应",
```

I am not sure which is better, `响应` or `回应`

---

use **响应**?
 I think it should be consistent with `"responseRequired": "需要响应"`

---

It's not about this specific line, but about the translation of "response" in general.
I checked the AWS and Azure official doc and seems like they both use `响应` for "response", so I think we are good here.